util/Order.cpp

```diff
@@ -104,7 +104,9 @@ void RenameOrder::ExecuteImpl() const {
 // CreateFleetOrder
 ////////////////////////////////////////////////
 NewFleetOrder::NewFleetOrder() :
-    Order()
+    Order(INVALID_OBJECT_ID),
```

Comment:
the Order constructor parameters is an empire id, not an object id, and the default is already to use ALL_EMPIRES